python/cudf/cudf/core/dataframe.py

```diff
@@ -4985,6 +4985,262 @@ def _setitem_with_dataframe(input_df, replace_df, input_cols=None, mask=None):
                 input_df.insert(len(input_df._data), col_1, replace_df[col_2])
 
 
+def to_datetime(
```

Comment:
Lets put this in `core/tools/datetimes.py` to match pandas: https://github.com/pandas-dev/pandas/blob/0.25.x/pandas/core/tools/datetimes.py#L563-L798

Reply:
Sure, done 👍